Fix call to create_load_global

aorenste opened the pull request:
There is no version of create_load_global() that takes three parameters - any use of this function will fail. I think this is probably the correct fix.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #145554
* __->__ #145553
* #145552
* #145551



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

aorenste committed:
Fix call to create_load_global

[ghstack-poisoned]

aorenste committed:
Update on "Fix call to create_load_global"

There is no version of create_load_global() that takes three parameters - any use of this function will fail. I think this is probably the correct fix.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

aorenste commented:
@aorenste has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D68593082).

aorenste committed:
Update on "Fix call to create_load_global"

There is no version of create_load_global() that takes three parameters - any use of this function will fail. I think this is probably the correct fix.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

Differential Revision: [D68593082](https://our.internmc.facebook.com/intern/diff/D68593082)

[ghstack-poisoned]

aorenste committed:
Update on "Fix call to create_load_global"

There is no version of create_load_global() that takes three parameters - any use of this function will fail. I think this is probably the correct fix.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

Differential Revision: [D68593082](https://our.internmc.facebook.com/intern/diff/D68593082)

[ghstack-poisoned]

aorenste committed:
Update on "Fix call to create_load_global"

There is no version of create_load_global() that takes three parameters - any use of this function will fail. I think this is probably the correct fix.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

Differential Revision: [D68593082](https://our.internmc.facebook.com/intern/diff/D68593082)

[ghstack-poisoned]

aorenste committed:
Update on "Fix call to create_load_global"

There is no version of create_load_global() that takes three parameters - any use of this function will fail. I think this is probably the correct fix.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

Differential Revision: [D68593082](https://our.internmc.facebook.com/intern/diff/D68593082)

[ghstack-poisoned]

aorenste committed:
Update on "Fix call to create_load_global"

There is no version of create_load_global() that takes three parameters - any use of this function will fail. I think this is probably the correct fix.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

Differential Revision: [D68593082](https://our.internmc.facebook.com/intern/diff/D68593082)

[ghstack-poisoned]

anijain2305 commented on torch/_dynamo/variables/builtin.py:
```diff
@@ -717,11 +717,11 @@ def as_proxy(self):
         assert self.fn.__module__ == "builtins"
         assert name not in codegen.tx.f_globals, "shadowed global"
-        codegen.append_output(codegen.create_load_global(name, False, add=True))
+        codegen.append_output(codegen.create_load_global(name, add=True))
```
We should try to add a test that triggers this.

aorenste committed:
Update on "Fix call to create_load_global"

There is no version of create_load_global() that takes three parameters - any use of this function will fail. I think this is probably the correct fix.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

Differential Revision: [D68593082](https://our.internmc.facebook.com/intern/diff/D68593082)

[ghstack-poisoned]
